{{cookiecutter.python_name}}/.github/workflows/check-release.yml

```
@@ -53,8 +55,9 @@ jobs:
         uses: jupyter-server/jupyter_releaser/.github/actions/check-release@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+      {% endraw %}
       - name: Upload Distributions
         uses: actions/upload-artifact@v2
         with:
-          name: {{ cookiecutter.python_name }}-releaser-dist-${{ github.run_number }}
+          name: {{ cookiecutter.python_name }}-releaser-dist-${{ '{{' }} github.run_number {{ '}}' }}
```

Optional suggestion, I find that syntax easier to read

```suggestion
          name: {{ cookiecutter.python_name }}-releaser-dist-${{ '{{ github.run_number }}' }}
```